eth-types/src/lib.rs

```diff
@@ -476,14 +476,14 @@ impl<'de> Deserialize<'de> for GethExecStep {
 }
 
 /// Helper type built to deal with the weird `result` field added between
-/// `GethExecutionTrace`s in `debug_traceBlockByHash` and
+/// [`GethExecTrace`]s in `debug_traceBlockByHash` and
```

Comment:
Wondering the reason why you add `[ ]` here? Forgot to add `(eth_types::GethExecTrace)`? L486 is the same as well.

Reply:
We mark a term as a keyword with `[ ]`. The `cargo doc` checks whether the referenced term in the brackets exists. Let's say if we rename `GethExecTrace` to something else, it will raise a compilation error and remind us to update the name here.

 For `(eth_types::GethExecTrace)`, if the referenced struct is already in the same file, we can drop the import path. Cargo doc raises a warning if we add an unnecessary explicit link.

Reply:
Thanks for explanation! Learnt something new. If so, do we need to add `[ ]` for `debug_traceBlockByNumber`in L480?

Reply:
Note that `debug_traceBlockByNumber` is a name from Geth RPC API; it is not a variable defined in our codebase, so the Cargo doc knows nothing about it.